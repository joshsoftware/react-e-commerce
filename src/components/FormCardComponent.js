import React from 'react';
import { Card } from 'reactstrap';
import { PropTypes } from 'prop-types';

const FormCardComponent = ({ data }) => {
  return <Card>{data}</Card>;
};

export default FormCardComponent;
FormCardComponent.propTypes = {
  data: PropTypes.oneOfType([PropTypes.element, PropTypes.array])
};
