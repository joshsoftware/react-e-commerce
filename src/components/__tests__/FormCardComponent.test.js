import { render, cleanup } from '@testing-library/react';
import React from 'react';
import FormCardComponent from '../FormCardComponent';

afterEach(cleanup);

describe('FormCardComponent', () => {
  it('Must return FormCardComponent', () => {
    const { asFragment } = render(<FormCardComponent data={<h3>hello world</h3>} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
