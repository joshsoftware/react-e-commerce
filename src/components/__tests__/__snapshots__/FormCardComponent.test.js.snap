// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCardComponent Must return FormCardComponent 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <h3>
      hello world
    </h3>
  </div>
</DocumentFragment>
`;
